### ISSUE TITLE
Audit all current use of ParagraphsDangerousHtml/LinebreaksDangerousHtml and replace with Markdown where applicable

### ISSUE THREAD

**Issue by japsu** [con2/kompassi#356]
Use Markdown from [con2/components](https://con2.github.io/components/paragraphs-dangerous-html)

![Image](https://github.com/con2/kompassi/assets/109397/927cad20-c0c8-4a1e-b7f0-0beca460af88)

**Comment by akx** [comment]
WYSIWYG doesn't preclude using Markdown as the underlying format though!

**Comment by japsu** [comment]
No, it doesn't. I'd prefer a WYSIWYG editor that does not expose non-technical users to Markdown, though. Outline is fine, Obsidian is too Markdowny.

**Comment by akx** [comment]
I was thinking something like https://quilljs.com/ or https://mdxeditor.dev/ or https://v3.mantine.dev/others/rte/...

**Comment by japsu** [comment]
Audit done as part of the @con2/components Markdown migration (05ef57ec7, 09f7cdf3c, 129e57330, c0bd2354c, b5b813476): no uses of `ParagraphsDangerousHtml`/`LinebreaksDangerousHtml` remain in the V2 frontend; call sites render via `<Markdown>` from @con2/components.

One residual raw HTML render remains for V1 email bodies: `src/app/[locale]/profile/messages/page.tsx` (`dangerouslySetInnerHTML` on `message.bodyHtml`). That is V1-produced content outside this issue's scope; file separately if it should be sanitized.

Closing as done.
